public/docs/ts/latest/cookbook/i18n.jade

```diff
@@ -92,6 +95,79 @@ a#i18n-attribute
   The Angular extraction tool preserves both the _meaning_ and the _description_ in the translation source file
   to facilitiate contextually-specific translations.
 
+  As an alternative, if you want to translate some content without having to create a new element, you can use a `<ng-container>`
+  which doesn't get renderered in the DOM:
+
++makeExample('cb-i18n/ts/app/app.component.html', 'i18n-ng-container', 'app/app.component.html')(format=".")
+
+:marked
+  You can also use a comment to get the same result:
+
++makeExample('cb-i18n/ts/app/app.component.html', 'i18n-with-comment', 'app/app.component.html')(format=".")
+
+a#translating-attributes
+:marked
+  ## Translating attributes with _i18n-x_
+  Now you add an image to your html and since you are concerned with a11y, you add an `alt` tag to it:
+
++makeExample('cb-i18n/ts/app/app.component.1.html', 'i18n-alt', 'app/app.component.html')(format=".")
+
+:marked
+  Such attribute needs to be translated as well, and `Angular` provides you another attribute: `i18n-x` where you replace
+  the `x` with the name of the attribute to translate. So the previous example would end like:
```

Comment:
" So the previous example would end like" -> in order to translate the xxx on the yyyy tag from the previous ex, you would write: